src/main/java/pl/kubiczak/orrrg/test/orrrgtest/Hello.java

```diff
@@ -10,4 +10,13 @@ String getMessage(String name) {
 
     return "Hello " + name + "!";
   }
+
+  // this is for sonar to report new issues
+  // on a branch travis run build without Sonarqube Check: https://travis-ci.org/orrrg/test/builds/213231858
+  private void notUsedMethod(){
+    long timestmp = System.currentTimeMillis(); // some comment at the end of line
+    /*
+    boolean isThisCodeCommented = true;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)